### PR TITLE
Is integer a palindrome example.

### PR DESCRIPTION
**Pull request by harrisonbrock**

### CODE REVIEW

File: topics/go/algorithms/numbers/palindrome/palindrome.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+// Is checks if a integer is a Palindrome.
+func Is(input int) bool {
+
+	// Integer below zero cannot be palindrome.
+	if input < 0 {
+		return false
+	}
+
+	// A integer is a palindrome if it has the value between 0 and 9.
```
Comment (ardan-bkennedy) [inline topics/go/algorithms/numbers/palindrome/palindrome.go:13]: An integer ....

File: topics/go/algorithms/numbers/palindrome/palindrome.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	if input >= 0 && input < 10 {
+		return true
+	}
+
+	// Get the reverse integer.
```
Comment (ardan-bkennedy) [inline topics/go/algorithms/numbers/palindrome/palindrome.go:18]: Reverse the digits in the integer.

File: topics/go/algorithms/numbers/palindrome/palindrome.go [hunk 3]
```diff
@@ -5,17 +5,17 @@ import "github.com/ardanlabs/gotraining/topics/go/algorithms/numbers/reverse"
 // Is checks if a integer is a Palindrome.
 func Is(input int) bool {
 
-	// Integer below zero cannot be palindrome.
+	// An negative integer is not a palindrome.
```
Comment (ardan-bkennedy) [inline topics/go/algorithms/numbers/palindrome/palindrome.go:8]: A negative integer.

An when the next word starts with a vowel.

A when it doesn't.

Comment (harrisonbrock) [reply topics/go/algorithms/numbers/palindrome/palindrome.go]: Fixed

**Pull request merged**